app/service/data_svc.py

```diff
@@ -2,14 +2,19 @@
 from base64 import b64encode
 from collections import defaultdict
 from datetime import datetime
-
+from uuid import UUID, uuid4
+import logging, os
 import yaml
 
 
+
+
 class DataService:
 
-    def __init__(self, dao):
+    def __init__(self, dao, debug_level):
         self.dao = dao
+        self.logger = logging.getLogger('DataService')
```

Comment:
let's not pass the debug level into this class-- it will automatically be inherited by the global one set in the server:main function.